pkg/azuremetadata/azuremetadata.go

```diff
@@ -0,0 +1,127 @@
+package azuremetadata
```

Comment:
Would `azuremetadataclient` be a better name? WDYT?

Reply:
renamed